skeleton/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+add_library(librw_skeleton
+        glfw.cpp
+        skeleton.cpp
+        skeleton.h
+        win.cpp
+    )
+
+set_target_properties(librw_skeleton
+    PROPERTIES
+        PREFIX ""
```

Comment:
I don't think setting the prefixes to *nothing* is really required.
I think it's weird to create libraries without `lib` prefix'es on posix systems.
These library are not dl'opened, so their exact filenames should not matter.
(I did not run premake5, so I didn't compare)


https://cmake.org/cmake/help/latest/prop_tgt/PREFIX.html